frontend/packages/vkt/src/enums/api.ts

```diff
@@ -17,6 +17,7 @@ export enum APIEndpoints {
  * The respective backend enum is `APIExceptionType`.
  */
 export enum APIError {
+  EnrollmentAlreadyPaid = 'enrollmentAlreadyPaid',
```

Comment:
En oo ihan varma mut muistelin että nää pitäis olla samassa järjestyksessä parhaillaan APIExceptionTypejen kanssa jotta frontendissa näytetään sopiva virheilmoitus backendin palauttamaan virheeseen nähden? Ehkä tosin ei tarvii mut vois testata.

Reply:
Testasin tätä pikaisesti ja vaikuttaa toimivan vaikka järjestys eri.

`APIExceptionType.getCode()` muuntaa nuo stringeiksi jotka palautetaan frontilla. Frontissa `notifier.ts: getAPIError()` sitten tuon koodin perusteella valitsee sopivan `APIError` enumin.

Reply:
Jes. Tää tuli sitten jossain vaiheessa korjattua, aikanaan piti olla järjestys sama